examples/dreamcast/pvr/yuv_converter/Makefile

```diff
@@ -0,0 +1,17 @@
+# KallistiOS ##version##
+#
+# examples/dreamcast/pvr/Makefile
+# (c) 2023 Andy Barajas
```

Comment:
The copyright here shouldn't be Andy's, as you're the one adding the file. Either it's yours or nobody's, unless he sent you the file outside of here somehow. If that's the case (or it should be you instead), please fix the formatting to match the suggestion on the other Makefile.

Reply:
@gyrovorbis Put your name there

Reply:
```suggestion
# Copyright (C) 2023 Falco Girgis
```